### PR TITLE
feat: add Cloud Armor to web-app-protection example

### PR DESCRIPTION
**Pull request by mbuitragos**
## What?

* Enable Cloud Armor
* Creating Security Rules 
* Creating OWASP Rules

## Why?

Enable security features on the Load Balancer.

## Testing ?

Integration and Lint tests were performed.

## Anything Else?

Owasp config security are not functional yet.
Changes are yet to be made.

### CODE REVIEW

File: examples/web-app-protection-example/main.tf [hunk 1]
```diff
@@ -125,6 +123,97 @@ module "mig_r2" {
   ]
 }
 
+resource "random_id" "suffix" {
+  byte_length = 4
+}
+
+module "cloud_armor" {
```
Comment (gtsorbo) [inline examples/web-app-protection-example/main.tf:130]: Can you use the existing Cloud Armor module: https://github.com/GoogleCloudPlatform/terraform-google-cloud-armor

**Review by gtsorbo — CHANGES_REQUESTED** [review]

File: examples/web-app-protection-example/main.tf (hunk 1 above)
Comment (mbuitragos) [reply examples/web-app-protection-example/main.tf]: We have changed the use of the cloud armor feature to use the Google module as requested.

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**